classes/Language.php

```diff
@@ -1235,7 +1235,12 @@ public static function downloadLanguagePack($iso, $version, &$errors = [])
     public static function downloadXLFLanguagePack($locale, &$errors = [], $type = self::PACK_TYPE_SYMFONY)
     {
         $file = self::getPathToCachedTranslationPack($locale, $type);
-        $url = (self::PACK_TYPE_EMAILS === $type) ? self::EMAILS_LANGUAGE_PACK_URL : self::SF_LANGUAGE_PACK_URL;
+
+        $baseUrl = self::EMAILS_LANGUAGE_PACK_URL;
```

Comment:
No way to modify this URL ?

Reply:
It can be done, but it is outside of the scope of this PR.

Reply:
Really ?!